cpp/ql/src/Security/CWE/CWE-190/ArithmeticWithExtremeValues.ql

```diff
@@ -35,18 +35,18 @@ predicate isMinValue(MacroInvocationExpr mie) {
 
 class SecurityOptionsArith extends SecurityOptions {
   override predicate isUserInput(Expr expr, string cause) {
-    (isMaxValue(expr) and cause = "max value") or
-    (isMinValue(expr) and cause = "min value")
+    (isMaxValue(expr) and cause = "overflow") or
+    (isMinValue(expr) and cause = "underflow")
```

Comment:
I think the old strings match better what this predicate actually computes. Unless there's a good reason to change these strings, I suggest keeping them as they are and having both a `string cause` and a `string effect` in the query, matching them up in the `or` formulas. It looks strange now that cause and effect are conflated.

Reply:
Done.